__tests__/app.test.js

```diff
@@ -85,3 +86,45 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test("GET 200: should return an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
```

Comment:
I think this could be a little nicer. 
Maybe using something like toMatchObject({key: expect.any(Number)}) etc etc